README.md

```diff
@@ -48,6 +23,19 @@ dependencies:
       cmd: pip3 install scipy
 ```
 
+## Supported Languages
+Below is the list of supported languages and the corresponding versions.
+- gcc 7
+- g++ 7
+- python 3.7
+- java 8 & 11
+
+### Installation scripts
+- Every supported language and its version has an installation script stored in `scripts` directory.
+- The scripts are named as `<language_version>.sh`. Example - For language - java and version - 8, script name should be `java_8.sh`.
+- The installation scripts are shell scripts that hold commands to install the corresponding language and its version.
```

Comment:
this statement is redundant to the first.

Reply:
Removed